x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/steps/auth.tsx

```diff
@@ -21,87 +21,111 @@ import {
   useFormContext,
   useFormData,
 } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
-import { Field, TextField, PasswordField } from '@kbn/es-ui-shared-plugin/static/forms/components';
+import {
+  Field,
+  TextField,
+  PasswordField,
+  CardRadioGroupField,
+} from '@kbn/es-ui-shared-plugin/static/forms/components';
 import { fieldValidators } from '@kbn/es-ui-shared-plugin/static/forms/helpers';
+import { AuthType as CasesWebhookAuthType } from '../../../../common/constants';
 import * as i18n from '../translations';
+
 const { emptyField } = fieldValidators;
 
 interface Props {
   display: boolean;
   readOnly: boolean;
 }
 
+const basicAuthFields = (readOnly: boolean) => (
```

Comment:
Is there any plan in the following PRs to share this code between the Cases Webhook and the classic Webhook?

Reply:
Yes, I was planning to do it when I handle the UI for the SSL option here as I will probably need to do some changes in `connector_types/webhook` 😁